gravitee-apim-console-webui/src/management/settings/api-logging/api-logging.component.html

```diff
@@ -297,6 +297,69 @@ <h3>Temporal</h3>
             apiLoggingForm.get('messageSampling.temporal')?.errors?.defaultLowerThanLimit
           }}</mat-error>
         </ng-container>
+        <ng-container formGroupName="windowedCount">
+          <h3>Windowed count</h3>
+          <p>Samples a count of message during a sliding duration window.</p>
+          <p>
+            Format is COUNT/DURATION (duration should conform to ISO-8601 duration format). Example: <strong>2/PT15S</strong> will sample at
+            most 2 messages during a 15 secs sliding window.
+          </p>
+
+          <mat-form-field
+            [matTooltip]="providedConfigurationMessage"
+            [matTooltipDisabled]="!isReadonly('logging.messageSampling.windowedCount.default')"
+            class="api-logging__form__card__form-field"
+            data-testid="windowedCount_default"
+          >
+            <mat-icon
+              *ngIf="isReadonly('logging.messageSampling.windowedCount.default')"
+              class="api-logging__form__card__form-field__icon"
+              matPrefix
+              >lock
+            </mat-icon>
+            <mat-label>Default</mat-label>
+            <input matInput formControlName="default" />
+            <mat-error *ngIf="apiLoggingForm.get('messageSampling.windowedCount.default').hasError('required')"
+              >Default value is required
+            </mat-error>
+            <mat-error *ngIf="apiLoggingForm.get('messageSampling.windowedCount.default').hasError('invalidFormat')"
+              >The sampling value follow this format: COUNT/DURATION, COUNT>0, DURATION is ISO-8601 duration format
+            </mat-error>
+            <mat-error *ngIf="apiLoggingForm.get('messageSampling.windowedCount.default')?.errors?.defaultLowerThanLimit">{{
+              apiLoggingForm.get('messageSampling.windowedCount.default')?.errors?.defaultLowerThanLimit
+            }}</mat-error>
+          </mat-form-field>
+
+          <mat-hint>The limit is the minimum allowed period to sample</mat-hint>
+          <mat-form-field
+            [matTooltip]="providedConfigurationMessage"
+            [matTooltipDisabled]="!isReadonly('logging.messageSampling.windowedCount.limit')"
+            class="api-logging__form__card__form-field"
+            data-testid="windowedCount_limit"
+          >
+            <mat-icon
+              *ngIf="isReadonly('logging.messageSampling.windowedCount.limit')"
```

Comment:
I realize we are not trying to do any major refactoring, but I have been advised to use `@if` syntax instead of `*ngIf`. So doing something like 
```
@if (isReadonly('logging.messageSampling.windowedCount.limit')) {
  <mat-icon
    class="api-logging__form__card__form-field__icon"
    matPrefix
    >lock
  </mat-icon>
}
```
in place is preferred by the newer Angular syntax.